kube/src/config/mod.rs

```diff
@@ -30,7 +30,9 @@ pub struct Config {
     /// Whether to accept invalid ceritifacts
     pub(crate) accept_invalid_certs: bool,
     /// The identity to use for communicating with the kubernetes API
-    pub(crate) identity: Option<reqwest::Identity>,
+    pub(crate) identity: Option<Vec<u8>>,
```

Comment:
Rather than masking the type (and risking introducing Der/Pem errors) rhere, we could manually implement Clone instead?

Reply:
What would that manual implementation do for the identity field?

Reply:
well, it would have to reach into the identity and extract the buffer like you've done, but at least it'd be confined to that one function.

Reply:
I don't think we can get the underlying buffer from the identity once we have a `Identity` type so we'd have to get a loader again and load the identity which is not very performant